Index: src/main/java/org/ozea/product/controller/ProductController.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.ozea.product.controller;\n\nimport org.ozea.product.dto.ProductDto;\nimport org.ozea.product.service.ProductService;\nimport org.springframework.web.bind.annotation.*;\n\nimport java.util.List;\n\n@RestController\n@RequestMapping(\"/api/products\")\npublic class ProductController {\n\n    private final ProductService productService;\n    public ProductController(ProductService productService) {\n        this.productService = productService;\n    }\n\n    @GetMapping\n    public List<ProductDto> list() {\n        return productService.findAll();\n    }\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/org/ozea/product/controller/ProductController.java b/src/main/java/org/ozea/product/controller/ProductController.java
--- a/src/main/java/org/ozea/product/controller/ProductController.java	(revision e75aeb290280d23a4f4446bbe49c39106ba090c4)
+++ b/src/main/java/org/ozea/product/controller/ProductController.java	(date 1762882253577)
@@ -1,19 +1,18 @@
 package org.ozea.product.controller;
 
+import lombok.RequiredArgsConstructor;
 import org.ozea.product.dto.ProductDto;
 import org.ozea.product.service.ProductService;
 import org.springframework.web.bind.annotation.*;
 
 import java.util.List;
 
+@RequiredArgsConstructor
 @RestController
 @RequestMapping("/api/products")
 public class ProductController {
 
     private final ProductService productService;
-    public ProductController(ProductService productService) {
-        this.productService = productService;
-    }
 
     @GetMapping
     public List<ProductDto> list() {
